src/esockd_server.erl

```diff
@@ -103,6 +129,26 @@ handle_call({init, {Protocol, ListenOn}, Metric}, _From, State) ->
     true = ets:insert(?STATS_TAB, {{{Protocol, ListenOn}, Metric}, 0}),
     {reply, ok, State, hibernate};
 
+handle_call({get_listener_prop, ListenerRef, Name}, _From, State) ->
+    Value = erlang:get(?LPROP(ListenerRef, Name)),
```

Comment:
Q: why store it in PD?

Reply:
It's an attempt to ensure things are hot-upgradeable. Overall, it's not 100% hot-upgradeable, but close to it. This is one of the questions that I wanted to clarify: how hard I should try to keep code (and correspondingly, runtime state) ready for a hot upgrade in this PR?

Reply:
no need to consider hot-up in this pr.